project_planning/project.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    type = fields.Selection(
```

Comment:
Change to m2o in a separate module with:
Sequence (drag-able) , Code and name.
Add those values as default.
        [('ADM', 'Administration'),
+         ('DEV', 'Development'),
+         ('FS', 'Functional Specifications'),
+         ('PM', 'Project Management'),
+         ('QA', 'Quality Assurance'),
+         ('SAL', 'Sales'),
+         ('TR', 'Training'),
+         ('TS', 'Technical Specifications')],